### ISSUE TITLE
Compare to other presidents

### ISSUE THREAD

**Issue by jayrav13** [trumptax/trumptax.me#9]
Opening Issues per feedback on Twitter:

https://twitter.com/PhilGroce/status/836186496742879233

> @trumptaxme Suggestion: Add comparisons to other Presidents (BHO, GWB) for reference, otherwise it’s just another big number.
>
> - PhilGroce

**Comment by ksdorothy** [comment]
Agreed I think this would be a great idea. Then we would have a sense of the magnitude and abuse of this president.

**Comment by AdamFreemer** [comment]
Here's an idea.

Example Obama.
http://www.judicialwatch.org/press-room/press-releases/judicial-watch-new-obama-travel-costs-bring-eight-year-total-96-million/
$96,000,000 / (365*8) = ~$32,876 / day.

Trump
~$38,000,000 / (4 months [120 days]) = ~$320,000 / day.

A daily spending vs. Obama/Bush per day comparison could be a cool add?
